app/models/project_group.rb

```diff
@@ -54,6 +54,21 @@ def summary_or_none
     summary.blank? ? "[#{I18n.t("none")}]" : summary.to_s
   end
 
+  def scheduled_start_date
+    return @scheduled_start_date if defined?(@scheduled_start_date)
+    @scheduled_start_date = reload.children.map(&:scheduled_start_date).compact.min
```

Comment:
Ok so I figured this out I think. I can describe it to you when you're in later. For now, just know that I moved the reloads into the spec where they belong and the specs are sill passing. So you can calm your nerves.